CSC8503CoreClasses/PlayerObject.cpp

```diff
@@ -127,7 +129,13 @@ void PlayerObject::UpdateObject(float dt) {
 			Debug::Print("Stunned", Vector2(45, 80));
 			break;
 		}
-		
+		const PlayerInventory::item equippedItem = GetEquippedItem();
+		//Handle Equipped Item Log
+		const std::string& itemName = mInventoryBuffSystemClassPtr->GetPlayerInventoryPtr()->GetItemName(equippedItem);
+		Debug::Print(itemName, Vector2(10, 80));
+		const std::string& usesLeft = "UsesLeft : " + to_string(mInventoryBuffSystemClassPtr->GetPlayerInventoryPtr()->GetItemUsesLeft(mPlayerID, mActiveItemSlot));
+		Debug::Print(usesLeft, Vector2(10, 85));
+		Debug::Print(to_string(mGameWorld->GetMainCamera().GetYaw()), Vector2(54, 90));
```

Comment:
Why are we printing out the player rotation?

Reply:
It was for debug purposes, can be turned off w/ DEBUG_MODE = false